ovnnbimp_test.go

```diff
@@ -475,4 +475,73 @@ func TestLoadBalancer(t *testing.T) {
 		t.Fatalf("error: lb deletion not done, total:%v", len(lb))
 	}
 	t.Logf("LB deletion done")
+	// test adding lb to lr and lswitch
+	ocmd, err = ovndbapi.LBAdd("lb2", "192.168.0.20:80", "tcp", []string{"10.0.0.21:80", "10.0.0.22:80"})
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = ovndbapi.Execute(ocmd)
+	if err != nil {
+		t.Fatalf("Adding LB OVN failed with err %v", err)
+	}
+	t.Logf("Adding LB lb2 to LSwitches %s", LSW)
+	ocmd, err = ovndbapi.LSLBAdd(LSW, "lb2")
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = ovndbapi.Execute(ocmd)
+	if err != nil {
+		t.Fatalf("Adding LB lb2 to LSwitch failed with err %v", err)
+	}
+	t.Logf("Adding LB lb2 to LSwitch %s Done", LSW)
+	t.Logf("List LB for LSwitch %s", LSW)
+	t.Logf("Deleting LB lb2 from LSwitch %s", LSW)
+	ocmd, err = ovndbapi.LSLBDel(LSW, "lb2")
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = ovndbapi.Execute(ocmd)
+	if err != nil {
+		t.Fatalf("Deleting LB lb2 from LSwitch failed with err %v", err)
+	}
+	t.Logf("Deleting LB lb2 from LSwitch %s Done", LSW)
+	// Add LB to router
+	t.Logf("Adding LB to LRouter %s", LR)
+	ocmd, err = ovndbapi.LRLBAdd(LR, "lb2")
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = ovndbapi.Execute(ocmd)
+	if err != nil {
+		t.Fatalf("Adding LB lb2 to LRouter failed with err %v", err)
+	}
+	t.Logf("Adding LB lb2 to LRouter %s Done", LR)
+	t.Logf("Deleting LB lb2 from LRouter %s", LR)
+	ocmd, err = ovndbapi.LRLBDel(LR, "lb2")
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = ovndbapi.Execute(ocmd)
+	if err != nil {
+		t.Fatalf("Deleting LB lb2 from LRouter failed with err %v", err)
+	}
+	t.Logf("Deleting LB lb2 from LRouter %s Done", LR)
```

Comment:
list it to make sure it is deleted successfully

Reply:
sure. Will do.

Reply:
same as above!